client/src/gen_expl/result_hierarchies.js

```diff
@@ -154,26 +157,22 @@ export function create_full_results_hierarchy({subject_guid, doc, allow_no_resul
               resources: get_resources(prog),
             }, 
           }));
+        }
+        // ^^ delete on drr17 exit
 
-        } else if ( (subject.is_first_wave && doc === 'drr17') || /dp/.test(doc) ){
-
-          return _.chain(subject.crsos)
-            .filter('is_cr')
-            .map(crso => ({
-              id: crso.guid,
-              isExpanded: false,
-              data: {
-                subject: crso,
-                type: 'cr',
-                name: crso.name,
-                resources: get_resources(crso),
-              }, 
-            }))
-            .value();
-
-        } 
-
-        break;
+        return _.chain(subject.crsos)
+          .filter('is_cr')
+          .map(crso => ({
+            id: crso.guid,
+            isExpanded: false,
+            data: {
+              subject: crso,
+              type: 'cr',
+              name: crso.name,
+              resources: get_resources(crso),
+            }, 
+          }))
+          .value();
```

Comment:
Does this work the same? Are first wave CRs really CRs or are they SOs?

Reply:
First wave CRs are true CRs.